PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+# What does this PR do? Why are doing this? References?
+[Tell us about your new feature, improvement, or fix! Fixes # (issue)]
+
+## Type of change
+[Please delete options that are not relevant.]
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## How Has This Been Tested?
```

Comment:
I don't think that this section is necessary. Reviewers need only look through the files to find the proper test cases (which they should be doing in the first place).

Reply:
I agree with you on short-run but might be useful when there are many people contributing.